### PR TITLE
Updating redhat_csp_download to use certified collection

### PR DESCRIPTION
**Pull request by hcherukuri**
Updating token in ansible.cfg

### CODE REVIEW

**Comment by hcherukuri** [comment]
Secrets are not available for the PR, so the CI will fail.  Please see on my fork branch: https://github.com/hcherukuri/jws-ansible-playbook/actions/runs/1495461824

**Review by gzaronikas — APPROVED** [review]
Looks good to me. Locally with a token, CI molecule test passes so I'm good with that. 
As a note to ourselves we should find a way to split the tests though in the future to avoid any GH CI breaks.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
     strategy:
       matrix:
         python_version: ["3.9"]
-    steps:
+    steps:
```
Comment (rpelisse) [inline .github/workflows/ci.yml:15]: @hcherukuri for the purpose of having clean commit, can you remove those extra whitespaces?

File: .gitignore [hunk 2]
```diff
@@ -1 +1 @@
-*.zip
\ No newline at end of file
+*.tar.gz
```
Comment (rpelisse) [inline .gitignore:1]: I would keep *.zip on top of *.tar.gz

**Review by rpelisse — CHANGES_REQUESTED** [review]
@hcherukuri just a few minor tweaks, otherwise LGTM!

File: .gitignore (hunk 2 above)
Comment (hcherukuri) [reply .gitignore:1]: Ok I will add .

**Review by rpelisse — APPROVED** [review]

**Pull request merged**